src/immutable.js

```diff
@@ -0,0 +1,9 @@
+export function removeKeysFromObject(object, keys) {
```

Comment:
I might consider renaming this to something like `util.js` since it's not doing real immutable stuff.

Reply:
Yes sure, the name is poor and can be confusing with https://facebook.github.io/immutable-js/